environment_win.yml

```diff
@@ -5,13 +5,14 @@ dependencies:
   - python=2.7
   - cachetools>=1.0.0
   - cgen
+  - coverage
   - enum34
-  - ffmpeg
+  - ffmpeg>=3.2.3,<3.2.6
   - flake8>=2.1.0
   - git
-  - m2w64-toolchain
   - jupyter
-  - matplotlib=2.0.2
+  - m2w64-toolchain
+  - matplotlib 2.0.2
```

Comment:
does this not need to be a `matplotlib=2.0.2`? Is the `=` not necessary on windows?

Reply:
Conda is relatively relaxed here: It will accept specification with and without a `=`.

In this particular file, the `=` was removed, because I copied the dependencies from the conda-forge recipe, where the notation matters. (Conda build does not accept an equal sign.)

Reply:
Conda is relatively relaxed here: It will accept specification with and without a `=`.

In this particular file, the `=` was removed, because I copied the dependencies from the conda-forge recipe, where the notation matters. (Conda build does not accept an equal sign.)